db/migrate/20220121034713_update_invalid_reminder_days_on_organizations.rb

```diff
@@ -0,0 +1,16 @@
+class UpdateInvalidReminderDaysOnOrganizations < ActiveRecord::Migration[6.1]
+  def up
+    safety_assured do
+      execute <<~SQL
+        UPDATE organizations
+        SET reminder_day = (CASE
+        WHEN deadline_day = 1 THEN 28
+        ELSE deadline_day - 1
+        END)
+          WHERE deadline_day IS NOT NULL
+          AND reminder_day IS NOT NULL
+          AND deadline_day = reminder_day;
+      SQL
+    end
```

Comment:
I'am a little rusty on reading SQL could you provide a comment on what this SQL is doing? I'am not sure what Invalid means in this case.

Reply:
Will add one! Were searching for any records with an identical deadline and reminder day, then setting the reminder day to the day before the deadline. If the deadline day is 1 then it wraps the reminder around to 28.

Reply:
Was this a valid use case in the past? Or was it an invalid one and we just never realized? Was there some report of the values being bad?

Reply:
I can't answer that exactly, but validation-wise it would have been possible to set both days to the same value.